pages/faq/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React from "react"
+
+export default function index() {
+  return (
```

Comment:
please do not define full `react` components in the `/pages` directory
- please move everything in the `return` to a new component in the `/components/` directory.